core/lib/spree/core/importer/product.rb

```diff
@@ -7,8 +7,8 @@ class Product
         def initialize(product, product_params, options = {})
           @product = product || Spree::Product.new(product_params)
 
-          @product_attrs = product_params
-          @variants_attrs = options[:variants_attrs] || []
+          @product_attrs = product_params.to_h
+          @variants_attrs = (options[:variants_attrs] || []).map(&:to_h)
```

Comment:
Hm, I still feel like the `.to_h` is more appropriate at the controller level, but I don't think it's important enough to hold this up.  I'm happy to see how this feels and readdress later if we feel like it.